test/arcade/patch/agent/module/PatchModuleStimulationTest.java

```diff
@@ -0,0 +1,94 @@
+package arcade.patch.agent.module;
+
+import java.lang.reflect.Field;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ec.util.MersenneTwisterFast;
+import arcade.core.util.Parameters;
+import arcade.patch.agent.cell.PatchCellCART;
+import arcade.patch.agent.cell.PatchCellTissue;
+import arcade.patch.agent.process.PatchProcessInflammation;
+import arcade.patch.sim.PatchSimulation;
+import arcade.patch.util.PatchEnums;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.verify;
+
+public class PatchModuleStimulationTest {
+
+    private PatchCellCART mockCell;
+
+    private PatchCellTissue mockTarget;
+
+    private PatchProcessInflammation mockInflammation;
+
+    private PatchModuleStimulation action;
+
+    private PatchSimulation sim;
+
+    private MersenneTwisterFast randomMock;
+
+    @BeforeEach
+    public final void setUp() {
+        mockCell = mock(PatchCellCART.class);
+        mockTarget = mock(PatchCellTissue.class);
+        mockInflammation = mock(PatchProcessInflammation.class);
+        sim = mock(PatchSimulation.class);
+        randomMock = mock(MersenneTwisterFast.class);
+        Parameters parameters = mock(Parameters.class);
+        doReturn(0.0).when(parameters).getDouble(any(String.class));
+        doReturn(0).when(parameters).getInt(any(String.class));
+
+        when(mockCell.getParameters()).thenReturn(parameters);
+        when(mockCell.getProcess(any())).thenReturn(mockInflammation);
+        when(mockInflammation.getInternal("granzyme")).thenReturn(1.0);
+        when(mockCell.getBoundTarget()).thenReturn(mockTarget);
+
+        action = new PatchModuleStimulation(mockCell);
```

Comment:
let's move this into the tests, let's be nicer to anyone who needs to read this later